pkg/oc/cli/cmd/set/volume.go

```diff
@@ -379,6 +333,24 @@ func (v *VolumeOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, out,
 	}
 	mapper, typer := f.Object()
 
+	numOps := 0
+	if v.Add {
+		numOps++
+	}
+	if v.Remove {
+		numOps++
+	}
+	if v.List {
+		numOps++
+	}
+
+	switch {
+	case numOps == 0:
+		v.List = true
+	case numOps > 1:
+		return errors.New("you may only specify one operation at a time")
```

Comment:
This looks like a validation error; keeping it in Validate seems right to me

Reply:
hm, since `numOps` is used to set the value of `v.List` to `true`, I'd be okay with still moving this to Validate and having that field completed there - still better than having all of the other completion logic that was in that method before. @soltysh for an opinion

Reply:
Let's leave this as is. I don't think we need to enforce every check be only in `Validate`.